apps/meteor/app/api/server/lib/getUploadFormData.ts

```diff
@@ -78,12 +80,13 @@ export async function getUploadFormData<
 				fileChunks.length = 0;
 				return returnError(new MeteorError('error-file-too-large'));
 			}
+			const mimeTypeFromFilename = mime.lookup(filename);
 
 			uploadedFile = {
 				file,
 				filename,
 				encoding,
-				mimetype,
+				mimetype: typeof mimeTypeFromFilename === 'string' ? mimeTypeFromFilename : mimetype,
```

Comment:
What does this variable being a string mean? 🤔 In which cases this will not be a string?

Reply:
`mime.lookup` may return a `string`, `string[]` or `undefined`
 - `undefined` in case no MIME type has been found for the extension in the DB
 - `string[]` will only happen in case we append an alternative MIME type to a extension (which we won't)
 So I'm just making sure we have the expected type here